.github/workflows/daily.yml

```diff
@@ -67,6 +67,8 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-python@v5
         with:
+          # TODO: Use Python 3.12. As of 2024-03-08, several third-party
+          # packages fail to install with Python 3.12.
           python-version: "3.10"
```

Comment:
We should probably reuse a common workflow for testing stubs, see https://docs.github.com/en/actions/using-workflows/reusing-workflows